bazel/test/arch_alias/README.md

```diff
@@ -0,0 +1,32 @@
+# arch_alias Tests
+
```

Comment:
please dont keep submitting voluminous bot docs - it overwhelms reviewers and detracts from the actual changes